src/ott/solvers/linear/sinkhorn_lr.py

```diff
@@ -94,16 +103,32 @@ def compute_reg_ot_cost(
     r: second factor of solution
     g: weights of solution
     ot_prob: linear problem
+    epsilon: Entropic regularization.
     use_danskin: if True, use Danskin's theorem :cite:`danskin:67,bertsekas:71`
       to avoid computing the gradient of the cost function.
 
   Returns:
     regularized OT cost, the (primal) transport cost of the low-rank solution.
   """
+
+  def ent(x: jnp.ndarray) -> float:
+    # generalized entropy
+    return jnp.sum(jsp.special.entr(x) + x)
```

Comment:
isn't this `- x` ?

Reply:
I think this should be correct - it's $-x log(x) + x$ and in Meyer's code, `np.sum(Q * (np.log(Q) - 1))` is used as the criterion (we substract our quantity later). But maybe it's the clearer to define as the latter case?

Reply:
Indeed the new obj in the unbalanced case when entropy is added is:
obj_{w.o entropy} + \esp np.sum(Q * (np.log(Q) - 1)) + ...

so if you define the generalized entropy it should be: entropy(x) + sum(x)

Reply:
thanks, made the mistake 👍 thanks!